docs/4.x/spa-tracking.md

```diff
@@ -37,6 +37,15 @@ _paq.push(['trackPageView']);
 
 Note: we recommend you use [Custom Dimensions](https://matomo.org/docs/custom-dimensions/) instead of Custom Variables as they will be deprecated in the future.
 
+### Resetting previously set custom dimensions
+
+Similar to Custom Variables, you also need to unset [Custom Dimensions](https://matomo.org/docs/custom-dimensions/) when changing the page as they would otherwise be tracked again.
```

Comment:
fyi @Findus23 looking at code they would also be sent with other actions (outlink, download, event) so wonder if they would need to be reset best after a `trackPageView` if it is a dimension in scope action. Like 

```
_paq.push(['trackPageView']);
_paq.push(['deleteCustomDimension', 1]);
```

Of course depends on the use case.

Reply:
That indeed depends on the use-case.

But on a "normal" page the custom dimension is also set when outlinks, etc. are tracked later on.
So I guess to stay as similar as possible to the "normal" use-case, I think deleting them on a route change is probably the best.